cluster_tools/Changelog.md

```diff
@@ -0,0 +1,32 @@
+# Change Log
+
+All notable changes to the cluser_tools library are documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/) `MAJOR.MINOR.PATCH`.
+For upgrade instructions, please check the respective *Breaking Changes* sections.
+
+## Unreleased
+[Commits](https://github.com/scalableminds/webknossos-libs/compare/v0.8.28...HEAD)
+
+### Breaking Changes
+- To download datasets, a recent webknossos server version is necessary (>= 21.12.0). webknossos.org is unaffected. [#510](https://github.com/scalableminds/webknossos-libs/pull/510)
```

Comment:
Does not look like cluster_tools thing to me? :thinking:

Reply:
argh, good catch. Actually, I even removed this, but afterwards I tested that the `make_changelog.py` worked with the new file and then did a hard reset to undo the changes (including this one) :scream: